avocado/utils/network/interfaces.py

```diff
@@ -612,6 +612,52 @@ def is_bond(self):
             LOG.debug(msg)
             return False
 
+    def is_veth(self):
+        """Check if interface is a Virtual Ethernet.
+
+        This method checks if the interface is a Virtual Ethernet or not.
+
+        rtype: bool
+        """
+        if not os.path.isfile(f"/sys/class/net/{self.name}/device/devspec"):
+            raise ValueError("Please provide valid inerface name")
```

Comment:
handle this gracefully may  raise NWException kind of way

Reply:
Done